kornia/augmentation/random_generator/base.py

```diff
@@ -46,10 +45,10 @@ def to(self, *args, **kwargs):
         self.set_rng_device_and_dtype(device=device, dtype=dtype)
         return self
 
-    def make_samplers(self, device: torch.device, dtype: torch.dtype) -> None:
+    def make_samplers(self, device: Device = None, dtype: Dtype = None) -> None:
```

Comment:
```suggestion
    def make_sampler(self, device: Device = None, dtype: Dtype = None) -> None:
```
as mentioned before to simplify this api so that is easy for anyone to create / customize the samplers

Reply:
/cc @shijianjian  @nitaifingerhut